lib/utils.js

```diff
@@ -435,3 +437,38 @@ function prettifyError ({ keyName, lines, eol, ident }) {
 
   return result
 }
+
+/**
+ * Deletes a specified property from a log object if it exists
+ *
+ * @param {object} log The log object to be modified.
+ * @param {string} property A string identifying the property to be deleted from
+ * the log object. Accepts nested properties delimited by a `.`
+ * e.g. `'prop1.prop2'`.
+ */
+function deleteLogProperty (log, property) {
+  const props = property.split('.')
+  const propToDelete = props.pop()
+
+  props.forEach((prop) => {
+    if (!Object.prototype.hasOwnProperty.call(log, prop)) {
+      return
+    }
+    log = log[prop]
+  })
+
+  delete log[propToDelete]
```

Comment:
the original implementation avoided modifying the existing object. I prefer allocating a new object in this case.

Reply:
I did think about this for a while but was not able to come up with a solution.
As the ignored keys can now be nested, we would need to perform a deep clone of the log object to avoid modifying the existing object. 
Is there a performance concern with doing a deep clone? Or if you have some other method in mind?
May I know what is the concern with modifying the original object? Thanks!

Reply:
I agree, do a deep clone before filtering. You can use `rdfc` for this.

Reply:
I assume it was a typo and you actually meant `rfdc`.
In the latest commit, the package was added and is used to clone the log object before filtering.